### DISCUSSION TITLE
有个网站AGE浏览页一直超时或者失败

### DISCUSSION THREAD

**Discussion by qingshujun** [xiaohucode/yidaRule#157] [Q&A] [answered]
有个动漫网站，搜索、列表页面这些都写好了，但是浏览页面视频请求，也参考其他网站，一直失败，有谁帮忙看看这个网站，播放页面视频正文要怎么获取。https://www.agedm.org/

**Comment by xiaohucode** [comment, marked as answer]
[AGE动漫.json](https://github.com/xiaohucode/yidaRule/files/15274217/AGE.json)
